com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -54,26 +50,19 @@ protected void OnEnable()
         {
             Debug.Assert(handNode == XRNode.LeftHand || handNode == XRNode.RightHand, $"HandVisualizer has an invalid XRNode ({handNode})!");
 
-            ControllerLookup[] lookups = FindObjectsOfType(typeof(ControllerLookup)) as ControllerLookup[];
```

Comment:
Could there be a case where `lookups` is empty or contains null objects? Do we still want to perform the switch statements?

Reply:
The existence of a physical xrController is not really important here, now that we're hooked into the tracking input action. We don't need to use `ControllerLookup `in this class.

This catch is very good, thanks for noticing Max!

Reply:
This line is completely removed now. The switch is only used to set `targetUsage`!